.circleci/config/jobs/website-docker-image.yml

```diff
@@ -0,0 +1,22 @@
+docker:
+  - image: circleci/buildpack-deps
+shell: /usr/bin/env bash -euo pipefail -c
+steps:
+  - checkout
+  - setup_remote_docker
+  - run:
+      name: Diff package-lock.json
+      command: |
+        echo 'export PACKAGE_LOCK_CHANGED=$(git diff --name-only $(git log --pretty=format:'%h' -n1 HEAD~1)...HEAD | grep -c website/package-lock.json)' >> $BASH_ENV
```

Comment:
Is the inner `git log` command necessary?  I see `git diff --name-only HEAD~1...HEAD` work too in my computer.  Also, I'd consider using `"` quotation here.  Given that the environment variable is only used once, below, have you considered assigning to variable directly without updating BASH_ENV?

Also, why should we inspect the last commit only?  The nomad team don't typically squash PRs before merging - so this might miss an actual dependences change.